### ISSUE TITLE
Support for a default view

### ISSUE THREAD

**Issue by peters** [publicclass/express-partials#6]
We have a lot of code that depends on having a default layout rendered depending on the what code module that is loaded. Since express has no concept of layouts anymore, we began looking for alternative implementations since rewriting is not an option, but a drop-in replacement is acceptable to us.

Then i found this great little library, and it will save us x amount of work. But how would one set a default layout, in 2.x we would do it like this:

```
app.set('view options', {
   layout: __magic_path + '/default_layout.ejs'
});
```

How can we do that using your middleware? :)

**Comment by slaskis** [comment]
Good catch, this is currently missing but it shouldn't be more than adding `app.get('view options').layout` to [this line](https://github.com/publicclass/express-partials/blob/master/index.js#L49).
